pages/login/index.vue

```diff
@@ -46,25 +49,52 @@
 </template>
 
 <script>
-import { useContext } from '@nuxtjs/composition-api'
-import { defineComponent } from '@vue/composition-api'
-
+import {
+  defineComponent,
+  useContext,
+  ref,
+  useRouter,
+} from '@nuxtjs/composition-api'
 export default defineComponent({
   setup() {
-    const { $toast } = useContext()
+    const { $axios, store, $toast } = useContext()
+    const router = useRouter()
+
+    const info = ref({
```

Comment:
`reactive` could have been a better choice but I have a feeling that you wrote it to not make it look like plag. It doesn't anyways code looks authentic could have gone with `reactive`